resources/views/components/link-preview-card.blade.php

```diff
@@ -0,0 +1,50 @@
+<div
+    id="link-preview-card"
+    data-url="{{ $url }}"
+    class="mx-auto mt-2 min-w-full group/preview" data-navigate-ignore="true"
+>
+@if ($data->has('html'))
+    @if ($data->get('site_name') === 'X (formerly Twitter)')
+        @php($tweetId = explode('/', $url))
+        <div data-tweet-id="{{ end($tweetId) }}" class="w-full overflow-hidden">
+            {!! $data->get('html') !!}
+        </div>
+    @else
+        <div class="w-full overflow-hidden">
+            {!! $data->get('html') !!}
```

Comment:
Is this safe?

Reply:
See above comment, has been made more safe.